src/main/java/org/mozilla/zest/core/v1/ZestRequest.java

```diff
@@ -326,15 +335,31 @@ public void clearCookies() {
 	}
 	
 	public void addCookie(String domain, String name, String value, String path, Date expiry, boolean secure) {
-		this.addCookie(new Cookie(domain, name, value, path, expiry, secure));
+		this.addCookie(new ZestCookie(domain, name, value, path, expiry, secure));
 	}
 	
-	public void addCookie(Cookie cookie) {
+	public void addCookie(ZestCookie cookie) {
 		this.cookies.add(cookie);
 	}
 
+	@Obsolete
```

Comment:
- Use `Deprecated` instead (same for following method).
 - Worth adding the JavaDoc pointing to the new method, also worth adding the version in which was deprecated, e.g.:
```Java
/**
 * @param cookie the cookie to add.
 * @deprecated (0.14) Use {@link #addCookie(ZestCookie)} instead.
 */
@Deprecated
public void addCookie(Cookie cookie) {
```
(Same comment for the other deprecated methods/constructors.)